packages/component-utils/package.json

```diff
@@ -0,0 +1,25 @@
+{
+  "name": "@packages/component-utils",
```

Comment:
how is this resolved into the package name that is published to `npm`?

Reply:
Still need to do a test run on a personal account to verify the setup we want

Reply:
Yeah I'm axing semver from being coupled to this release